src/interfaces/permissions-grant.ts

```diff
@@ -104,4 +106,34 @@ export class PermissionsGrant extends Message<PermissionsGrantMessage> {
       );
     }
   }
+
+  public static validateScope(permissionsGrantMessage: PermissionsGrantMessage): void {
+    const scope = permissionsGrantMessage.descriptor.scope;
+    if (scope.interface === DwnInterfaceName.Records) {
```

Comment:
Sanity: this line seems to read: "scope can be anything if interface is not `Records`", consider just remove, or add `else`.